src/main/java/com/khomishchak/apigateway/validator/RouterValidator.java

```diff
@@ -0,0 +1,19 @@
+package com.khomishchak.apigateway.validator;
+
+import org.springframework.http.server.reactive.ServerHttpRequest;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+@Component
+public class RouterValidator {
+
+    public static final List<String> openApiEndpoints = List.of(
```

Comment:
I would recommend u to use unmodifiable list here to ensure that the list remains unmodifiable, even if someone tries to modify it later unintentionally:

```
public static final List<String> openApiEndpoints = Collections.unmodifiableList(
        List.of("/auth/register", "/auth/login", "/users", "/users/authenticate", "/exchangers/used-currencies")
);
```

Reply:
List.of() creates immutable List as well